keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/SlackAppController.kt

```diff
@@ -34,12 +41,25 @@ class SlackAppController(
             .build()
           ctx.respond(response)
         }
+      } else if (req.payload.notificationType == "FULL_COMMIT_MODAL") {
+        val modal: View = commitModalCallbackHandler.buildView(req.payload)
+        val response = ctx.client().viewsOpen { r ->
+          r
+            .token(slackConfiguration.token)
+            .triggerId(req.payload.triggerId)
+            .view(modal)
+        }
+        if (!response.isOk) {
+          log.error("Failed to send slack callback: {}", response)
+        }
```

Comment:
Kind of a nit, but I think the code would be cleaner if we encapsulated all this logic in `CommitModalCallbackHandler` in a method like `handleCallback` that takes the request and context and returns a response.

Reply:
extracted both this and the mj stuff into a method on the callback handlers